cql3/functions/time_uuid_fcts.hh

```diff
@@ -62,6 +62,18 @@ make_now_fct() {
     });
 }
 
+static int64_t get_valid_timestamp(const data_value& ts_obj) {
+    auto ts = value_cast<db_clock::time_point>(ts_obj);
+    int64_t ms = ts.time_since_epoch().count();
+    auto nanos_since = utils::UUID_gen::make_nanos_since(ms);
+    if (!utils::UUID_gen::is_valid_nanos_since(nanos_since)) {
+        if (!utils::UUID_gen::is_valid_nanos_since(nanos_since)) {
```

Comment:
why is the same condition checked twice?

Reply:
hmm, this was unintentional.  Will fix.

Reply:
Fixed in v3